src/components/client/notifications/RepoNotificationSetting.tsx

```
@@ -45,10 +51,10 @@ export default function RepoNotificationSetting({
     },
     onMutate: async (payload: { types: string[] }) => {
       await queryClient.cancelQueries({ queryKey: ["repositories"] });
-      const previousData = queryClient.getQueryData<any[]>(["repositories"]);
-      queryClient.setQueryData<any[]>(["repositories"], (oldData) => {
+      const previousData = queryClient.getQueryData(["repositories"]);
+      queryClient.setQueryData(["repositories"], (oldData: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)